WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/GetPostsForTagWithCountUseCase.kt

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.ui.reader.repository.usecases
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.async
+import kotlinx.coroutines.withContext
+import org.wordpress.android.datasets.ReaderPostTable
+import org.wordpress.android.models.ReaderPostList
+import org.wordpress.android.models.ReaderTag
+import org.wordpress.android.modules.BG_THREAD
+import javax.inject.Inject
+import javax.inject.Named
+
+class GetPostsForTagWithCountUseCase @Inject constructor(
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
+) : ReaderRepositoryDispatchingUseCase(bgDispatcher) {
+    suspend fun get(readerTag: ReaderTag): Pair<ReaderPostList, Int> =
+            withContext(bgDispatcher) {
```

Comment:
We might want to use `coroutineContext` instead of `bgDispatcher` here. Wdyt?

Reply:
in commit 576c59f